src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionTest.cs

```diff
@@ -10,33 +11,39 @@
 using Microsoft.CodeAnalysis.CodeRefactorings;
 using Microsoft.CodeAnalysis.Editor.Implementation.Preview;
 using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.PickMembers;
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Test.Utilities;
 using Roslyn.Utilities;
 using Xunit;
-using Microsoft.CodeAnalysis.UnitTests;
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions
 {
     public abstract class AbstractCodeActionTest : AbstractCodeActionOrUserDiagnosticTest
     {
-        protected abstract CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace);
+        protected virtual CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace)
+            => throw new NotImplementedException();
```

Comment:
this approach allowed all the existing overrides to not have to be updated... but i could just go and update them all...